src/rocker/build/commands.go

```diff
@@ -931,45 +931,49 @@ func (c *CommandPush) Execute(b *Build) (State, error) {
 		return b.state, err
 	}
 
-	if !b.cfg.Push {
-		log.Infof("| Don't push. Pass --push flag to actually push to the registry")
-		return b.state, nil
-	}
-
-	image := imagename.NewFromString(c.cfg.args[0])
-
-	digest, err := b.client.PushImage(image.String())
-	if err != nil {
-		return b.state, err
+	var image *imagename.ImageName
+	var lines []string
+
+	if b.cfg.Push || b.cfg.ArtifactsPath != "" {
```

Comment:
This duplicates the logic a bit. Need think about how to make it more simple. Maybe, just don't do this condition and initialize it anyway, even if we will not need it.